Cargo.toml

```diff
@@ -122,6 +122,9 @@ bytemuck = "1.7"
 futures-lite = "1.11.3"
 crossbeam-channel = "0.5.0"
 
+# Example crates
+examples_utils = { path = "examples_utils", version = "0.8.0-dev" }
```

Comment:
The main reason why it is named `example_utils` is so that in the repo it would be near the `examples` dir. But yeah, on crates.io it would look weird. Should I still change it?